src/gltf-component-mappings.js

```diff
@@ -131,13 +131,18 @@ AFRAME.GLTFModelPlus.registerComponent("media", "media", (el, componentName, com
 });
 
 function mediaInflator(el, componentName, componentData, components) {
+  let isControlled = true;
```

Comment:
Seems like you are only using this within the conditional, why define it as a let out here instead of a const inside like before?

Reply:
its used later on line 173